### PR TITLE
Support project-local ini files

### PR DESCRIPTION
**Pull request by strub**
EasyCrypt now reads project-local files for getting its configuration settings. This configuration file takes precedence over the system-level configuration file.

The project-local configuration file is named 'easycrypt.project'.  It is identical to the system-level configuration file.

When compiling a file, the configuration file is searched upward from the target file directory. When ran in interactive mode, the configuration file is searched upward from the current working directory.

Note that when ProofGeneral starts EasyCrypt, the working directory is set to the directory of the focused file.

### CODE REVIEW

File: src/ec.ml [hunk 1]
```diff
@@ -479,4 +527,6 @@ let main () =
     raise e
 
 (* -------------------------------------------------------------------- *)
-let () = main ()
+let () =
+  Format.eprintf "%s" (Unix.getcwd ());
```
Comment (bgregoir) [inline src/ec.ml:531]: Is it intentional ?

Comment (bgregoir) [reply src/ec.ml:531]: The eprintf in the line 531?

**Review by bgregoir — APPROVED** [review]

File: src/ec.ml (hunk 1 above)
Comment (strub) [reply src/ec.ml:531]: Of course not :)

**Review by fdupress — APPROVED** [review]
lgtm

**Pull request merged**